insecure/orchestrator/network.go

```diff
@@ -194,11 +197,13 @@ func (on *Network) processIngressMessage(message *insecure.IngressMessage) error
 	on.orchestratorMutex.Lock()
 	defer on.orchestratorMutex.Unlock()
 
+	ingressEventID := flow.HashToID(hash.DefaultHasher.ComputeHash(message.Payload))
```

Comment:
Please also use [EventId](https://github.com/onflow/flow-go/blob/master/network/p2p/network.go#L528-L528) method for this `ingressEventID` computation.

Reply:
https://github.com/onflow/flow-go/pull/3580/commits/3c84a08a7331610b83cc9981f90f9c24acd326ed